docs/host-parameters-validation.md

```diff
@@ -0,0 +1,19 @@
+# HostParametersValidation
+
+## Description
+
+This rule is to valiate the parameters in the 'x-ms-parameterized-host' to ensure they are following below rules:
```

Comment:
This is to validate if parameters in the 'x-ms-parameterized-host' follow the following rules: